src/xenia/base/clock_posix.cc

```diff
@@ -15,17 +15,17 @@
 namespace xe {
 
 uint64_t Clock::host_tick_frequency() {
-  timespec res;
-  clock_getres(CLOCK_MONOTONIC_RAW, &res);
-
-  return uint64_t(res.tv_sec) + uint64_t(res.tv_nsec) * 1000000000ull;
+  // Even if our resolution is less, CLOCK_MONOTONIC_RAW will always return
+  // ns values. Most of the time the resolution is pretty close to 1ns anyway
+  // due to the presence of hr timers, so just pass along ns ticks
+  return 1000000000ull;
```

Comment:
Even though I can understand your reasoning - is there a specific reason you decided to hardcode this value rather than use `clock_getres`?

Reply:
Good question. So the clock_getres always returns 1ns in Linux even if the clock doesn't actually support that resolution, so I think my thoughts at the time were why bother? But then again why did I bother to change it in the first place? I don't remember... I will revert this.

Reply:
I noticed that the original frequency calculation was technically wrong even though it produced the right value, so I fixed that.